projects/kit/components/slider/slider.style.less

```diff
@@ -1,12 +1,129 @@
-.dot:first-child {
-    display: none;
+/*
+*** Thumb – this is an element on the track that the user can move around to select varying range values.
+*** Track – this is the part of the slider that the thumb runs along.
+*** Ticks - tick marks along the length of the track.
+*/
+
+@thumb-heights: {
+    @s: 0.5rem;
+    @m: 1rem;
+};
+@track-height: 0.125rem;
+@ticks-thickness: 0.25rem;
+
+@track-color: var(--tui-base-03);
+@thumb-color: currentColor;
+
+.ticks() {
+    background: @track-color;
+    background: linear-gradient(to right, var(--tui-base-07) @ticks-thickness, @track-color @ticks-thickness) repeat-x;
+    background-size: var(--tui-slider-step-percentage) @track-height;
+    background-position: 0 center;
+}
+
+.customize-track() {
+    height: @track-height;
+    border-radius: var(--tui-radius-m);
+}
+
+.customize-thumb(@size) {
+    appearance: none;
+    background-color: @thumb-color;
+    border: none;
+    border-radius: 50%;
+    height: @thumb-heights[ @@size];
+    width: @thumb-heights[ @@size];
+
+    :not(:disabled)& {
+        cursor: ew-resize;
+    }
+
+    :focus& {
+        box-shadow: 0 0 0 2px inset var(--tui-focus);
+    }
+}
+
+/* Vertically centers thumb on the track (webkit only) */
+.vertically-align-thumb(@size) {
+    margin-top: (@track-height / 2) - (@thumb-heights[ @@size] / 2);
+}
+
+:host {
+    display: block;
+    width: 100%;
+    color: var(--tui-primary); /* See currentColor inside this file */
+    cursor: pointer;
+
+    &:disabled {
+        opacity: var(--tui-disabled-opacity);
+        cursor: auto;
+    }
 }
 
-.wrapper {
-    border-left: none;
+:host:not(._old-edge) {
+    appearance: none;
+    height: @track-height;
+    padding: 0.4375rem 0; /* To catch click events nearby THIN input's track */
+    background: @track-color;
+    background-clip: content-box; /* To clip filling of paddings */
+
+    &:focus {
```

Comment:
No need to put it in `:focus`, just leave at `:host`. It's like people often put `cursor: pointer` in `:hover` :D

Reply:
fixed